assets/js/modules/todo-module.js

```diff
@@ -37,6 +33,22 @@ Todo = function () {
 		}
 	}();
 	var table = function(){
+		function tables(val){
+			var table = $(".maintable").DataTable({});
+			$.ajax({
```

Comment:
add your ajax request inside datatable.
`$('.table).dataTable({
    // other fields you might need
    ajax: {
        // TODO
    }
});